chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ActivityController.java

```diff
@@ -502,6 +508,7 @@
             notes = "Users must specify 'start' and 'end' parameters to restrict the date range of the results.")
     @RequestMapping(value = "/pending_certified_products", method = RequestMethod.GET,
             produces = "application/json; charset=utf-8")
+    @PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_ACB')")
```

Comment:
I'm not sure if we allow ROLE_ADMIN to see pending Listings. If we don't currently, then we probably shouldn't with this line. If we do, but the UI restricts it, then this is fine.